templates/custom/vesta/vesta.html.twig

```diff
@@ -54,6 +54,9 @@
               <i class="icon-normalphone" aria-hidden="true"></i>
               <span>
                 <a href="tel:{{ landline.value }}" alt="Person tel">{{ landline.value }}</a>
+                {% if landline.note is not empty %}
```

Comment:
Not landline.not|render|striptags|trim ?

Reply:
`landline.note` returns a string, as defined in `Gent\Services\Vesta\Value\ContactNumber`, so just checking for an empty string should do 😉

Reply:
Oh it doesn't pass through Drupal, ok :)